lib/pages/trade/desktop_trade.dart

```diff
@@ -50,668 +50,684 @@ class _DesktopTradeState extends State<DesktopTrade> {
     final tokenContainerWid = wid * 0.95;
     final amountBoxAndMaxButtonWid = tokenContainerWid * 0.5;
 
-    return BlocConsumer<TradePageBloc, TradePageState>(
-      listenWhen: (_, current) => current.status == BlocStatus.error,
+    return BlocListener<WalletBloc, WalletState>(
       listener: (context, state) {
-        if (state.failure is NoSwapInfoFailure) {
-          context.showWarningToast(
-            title: 'Action Error',
-            description: 'No swap info found',
-          );
+        if (state.isWalletConnected || state.isWalletDisconnected) {
+          context.read<TradePageBloc>().add(WatchAppDataChangesStarted());
+          _tokenFromInputController.clear();
+          _tokenToInputController.clear();
```

Comment:
Add some debug print statements for the remaining states

Reply:
Done